pkg/sippyserver/pr_commenting_processor.go

```diff
@@ -127,6 +127,9 @@ type RiskAnalysisEntryList []RiskAnalysisEntry
 func (r RiskAnalysisEntryList) Len() int      { return len(r) }
 func (r RiskAnalysisEntryList) Swap(i, j int) { r[i], r[j] = r[j], r[i] }
 func (r RiskAnalysisEntryList) Less(i, j int) bool {
+	if r[i].Value.RiskLevel.Level == r[j].Value.RiskLevel.Level {
+		return r[i].Value.Name > r[j].Value.Name
```

Comment:
can you pls explain why this is necessary?  if the risk levels are identical, how is sorting by name a meaningful ordering?

Reply:
We want the order of the comment text to always be the same so we can compare the comment update with the existing comment and be sure we are writing a new comment (not a second processing of the existing run which can happen).  So the order that the failed jobs appears in the comment is important along with ranking based on the Level.